examples/bar_polar.py

```diff
@@ -0,0 +1,75 @@
+from dash import Dash, dcc, html, Input, Output
+import pandas as pd
+from datetime import date
+import plotly.express as px
+
+filepath = "https://raw.githubusercontent.com/plotly/datasets/master/2011_us_ag_exports.csv"
+df = pd.read_csv(filepath)
+
+state_list = list(df["state"].unique())
+
+chosen_list = ["Alabama", "Alaska", "Arizona"]
+
+app = Dash(__name__)
+
+app.layout = html.Div([
+    html.Div([
+        html.H1(f'Polar Charts of U.S. Agricultural Exports, 2011',
+                style={'textAlign': 'center'})]),
+    html.Div([
+        html.H2('Choose the radius scale (absolute or logarithmic), then choose states:',
+                style={"margin-left": "5em"}),
+        html.Div(
+            [dcc.RadioItems(
+                id="bar-polar-app-x-radio-items",
+                options=["Absolute", "Logarithmic"],
+                value="Absolute"
+            )
+            ], style={"width": "50%",
+                      'font-size': 20,
+                      "marginLeft": "6em"}
+        ),
+        html.Div([html.Br()]),
+        html.Div([dcc.Dropdown(
+            id='bar-polar-app-x-dropdown',
+            value=state_list[:6],
+            options=state_list,
+            multi=True,
+            style={'font-size': 16,
+                   "width": "90%",
+                   "marginLeft": "3.8em"}
+        ),
+            dcc.Graph(id='bar-polar-app-x-graph',
+                      style={
+                          "margin-left": "5em", })
+
+        ], style={"width": "90%"}
+        )
+    ], style={"width": "100%"}
+    )
+]
+)
+
```

Comment:
This layout looks nice on some screen sizes, but looks unbalanced on others.  It might be better to simplify the layout to something like this:

```python
app.layout = html.Div(
    [
        html.H1(
            f"Polar Charts of U.S. Agricultural Exports, 2011",
            style={"textAlign": "center"},
        ),
        html.Div("Choose the radius scale:"),
        dcc.RadioItems(
            id="bar-polar-app-x-radio-items",
            options=["Absolute", "Logarithmic"],
            value="Logarithmic",
        ),
        html.Br(),
        html.Div("Choose States:"),
        dcc.Dropdown(
            id="bar-polar-app-x-dropdown",
            value=state_list[:6],
            options=state_list,
            multi=True,
        ),
        dcc.Graph(id="bar-polar-app-x-graph"),
    ],
    style={"margin": "6em", "fontSize": 20},
)

```

Reply:
Love it! And the code looks much cleaner too, thank you so much!